### ISSUE TITLE
The "network request log" panel opens automatically at installation.

### ISSUE THREAD

**Issue by Percherie** [uBlock-LLC/uBlock#1764]
Good morning,

Within the framework of a deployment in a school, user profiles are fixed and lightened to the maximum. There is therefore no local profile for Firefox and it is automatically generated if the user uses the browser.

The uBlock extension is therefore placed in the Firefox installation folder"<firefox>/distribution/extension".

Problem, with each installation, uBlock opens the "network request log" side panel. How can we prevent it from opening?

By the way, why did you post it by default? This profoundly changes the user interface and only experienced users can use this console.

Translated with www.DeepL.com/Translator

**Comment by krystian3w** [comment]
You added question in wrong place... https://www.reddit.com/r/uBlockOrigin/ this is should better if have reddit account.

### Workaround

Hide with userChrome.css?

```css
vbox#sidebar-box[sidebarcommand="_UUID~ADDON_-sidebar-action"] {
   display: none !important;
}

vbox#sidebar-box[sidebarcommand="ublock0_raymondhill_net-sidebar-action"] {
   display: none !important;
}
```

`UUID~ADDON` version use when `ublock0_raymondhill_net` does not work
or using old µBlock 0.9.5.x (µBlock - have side panel...:question:).

**Comment by Percherie** [comment]
It is interesting to use userChrome.css but it requires that a user profile exists first.

The users of the domain have a fixed and lightened profile between 3 and 6 Mo. The Firefox profile can be reduced to 400 KB but the extensions are too big.

I am studying an alternative solution which consists in configuring a Firefox profile generated during the first use of FireFox. 
For that I prepare a profile with the CCK2 extension: https://mike.kaply.com/
I place this profile in the Firefox installation folder
Then I place the extensions in the <Firefox folder>/distribution/extension

I abandoned the option of a profile placed in c:/users/default because I use fixed profiles that are dropped at each connection.

Is there a way to generate a userChrome.css imposed for new users? Or a modification to the extension so that the panel is not displayed immediately after installation?

Translated with www.DeepL.com/Translator

**Comment by krystian3w** [comment]
Fixed in uBO 1.16.6 https://github.com/gorhill/uBlock/releases/tag/1.16.6

https://bugzilla.mozilla.org/show_bug.cgi?id=1459007

**Comment by krystian3w** [comment]
Fixed in uBO 1.16.6 https://github.com/gorhill/uBlock/releases/tag/1.16.6 (delete/disable sidebar logger uBO from stable build).
